### ISSUE TITLE
[REQUEST] Landing lights click spot

### ISSUE THREAD

**Issue by zannu82** [flybywiresim/a32nx#1897]
**Is your feature request related to a problem? Please describe.**

**Describe the feature you'd like to see implemented**

I like ti have a clickspot between the two landing lights. In this way i have the possibility to swith on and off clicking only in one point. The same function is present in the falab addon. 

**References**

**Additional context**

Discord username (if different from GitHub):
zannu

**Comment by Snapmatics** [comment]
Not possible until we get the model in.

**Comment by zannu82** [comment]
thanks but what does it mean?

**Comment by Benjozork** [comment]
@Snapmatics

Correction: this is doable, just not decided on yet.

**Comment by zannu82** [comment]
Ah ok. Thanks for your answer

**Comment by VeIocityy** [comment]
Bumping, the Fenix has something similar (https://gyazo.com/e7eff1998c05ba4cd1a6b3a8c265f9ca). Would like to see something like this implemented. Makes flows a lot better.

**Comment by VeIocityy** [comment]
bump

**Comment by tracernz** [comment]
We will consider this. It will need some 3d model changes so won’t happen overnight.

**Comment by VeIocityy** [comment]
even though it's just a "switch" inbetween both of the landing lights?

**Comment by arkob145** [comment]
yes, please. doesn't even need to be near the landing lights. Even of the MCP or the chrono button will be fine.

**Comment by alepouna** [comment]
> even though it's just a "switch" inbetween both of the landing lights?

It looks like a simple thing on the surface but it may not be, especially if it involves the 3d model 😉

**Comment by VeIocityy** [comment]
Understood. I'll leave it to the smart people.
